topics/tutorials/custom_language_support/structure_aware_navbar.md

```diff
@@ -0,0 +1,44 @@
+[//]: # ("title: 15. Structure Aware Navigation Bar")
```

Comment:
Please add an entry in intellij-sdk-docs/ijs.tree file, otherwise, it won't be listed in the page structure.